docs/maestro_config.md

```diff
@@ -34,16 +34,16 @@ network:
   hw_addr: "{{ARCH_ETHERNET_MAC}}"
 ```
 
-Breakdown:
-* `if_name` - REQUIRED. Name of the interface Maestro will modify
-* `existing` - OPTIONAL. Tells Maestro to `replace` or `override` the existing saved interface
-* `clear_addresses` - REQUIRED. Clears any existing addresses assigned to the interface before setting up the specified addresses
-* `dhcpv4` - REQUIRED. `false` for static interfaces
-* `ipv4_addr` - REQUIRED. IP address to assign to the interface
-* `ipv4_mask` - REQUIRED. IP mask to use for the subnet
-* `hw_addr` - MAC address to use for the interface
+Parameters:
+* `if_name` - REQUIRED. Name of the interface Maestro modifies.
+* `existing` - OPTIONAL. Tells Maestro to `replace` or `override` the existing saved interface.
+* `clear_addresses` - REQUIRED. Clears any existing addresses assigned to the interface before setting up the specified addresses.
+* `dhcpv4` - REQUIRED. `false` for static interfaces.
+* `ipv4_addr` - REQUIRED. IP address to assign to the interface.
+* `ipv4_mask` - REQUIRED. IP mask to use for the subnet.
+* `hw_addr` - MAC address to use for the interface.
 
-### DHCP interface:
+### DHCP interface
```

Comment:
The examples for static interface and DHCP interface seem to be identical. Is this correct?

Reply:
Thank You for pointing this out. It is not correct. I did notice this a few days back and have corrected it. My changes are already in dev branch which will be merged into master in the next release.